### DISCUSSION TITLE
Is SOZI dead?

### DISCUSSION THREAD

**Discussion by RafaelLinux** [senshu/Sozi#701] [General]
I tried SOZI 2 years ago. I see now that currently is not updated. Is SOZI dead?

**Comment by ejvindh** [comment]
Latest release is from July 2023 -- doesn't sound dead to me ;)
https://github.com/sozi-projects/Sozi/releases/tag/v23.07

**Comment by RafaelLinux** [comment]
I saw SRC last modify was from 2 years ago, that's why I thought project was dead. I'll try latest AppImage.

Thank you!!

**Comment by bidinou** [comment]
Hi ! I love Sozi, I keep getting back to it after several years. Definitely hope it's not dead :-) 
Cheers !
